tasks/agent6-linux.yml

```diff
@@ -10,6 +10,13 @@
     dest: /etc/datadog-agent
     state: directory
 
+- name: Create system-probe configuration file
```

Comment:
Is the system prob only available one linux ? Do we plan to have support for Windows at some point ?

Please move this after line 52 (before or after `Create trace agent configuration file`). We want to create the datadog.yaml first (in case it fails the error raise to the customer will be about the main agent).

Reply:
> Is the system prob only available one linux ? 

Yes

> Do we plan to have support for Windows at some point ?

I believe so.